open_spiel/python/tests/sampled_stochastic_games_test.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 DeepMind Technologies Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import pickle
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import numpy as np
+import pyspiel
+
+# All games with kSampledStochastic chance mode.
+SPIEL_SAMPLED_STOCHASTIC_GAMES_LIST = [
+    g for g in pyspiel.registered_games() if g.default_loadable
+    and g.chance_mode == pyspiel.GameType.ChanceMode.SAMPLED_STOCHASTIC
+]
+assert len(SPIEL_SAMPLED_STOCHASTIC_GAMES_LIST) >= 2
+
+
+class SampledStochasticGamesTest(parameterized.TestCase):
+    def random_playout(self, state):
+        np.random.seed(0)
+        while not state.is_terminal():
+            state.apply_action(np.random.choice(state.legal_actions()))
+        return state
```

Comment:
Can you make this a free function? And add it to python/tests/test_utils.py? This is something that should be useful to a bunch of other tests.

Also, can you make seed an argument with a default value of 0?